orator_development.py

```diff
@@ -1,10 +1,17 @@
+import os
+
+DB_HOSTNAME = os.getenv("DB_HOSTNAME")
+DB_NAME = os.getenv("DB_NAME")
+DB_USERNAME = os.getenv("DB_USERNAME")
+DB_PASSWORD = os.getenv("DB_PASSWORD")
+
 DATABASES = {
-    'development': {
-        'driver': 'postgres',
-        'host': 'localhost',
-        'database': 'neo-data',
-        'user': 'neo',
-        'password': 'neoempresarial',
-        'prefix': ''
+    "production": {
+        "driver": "postgres",
+        "host": DB_HOSTNAME,
+        "database": DB_NAME,
+        "user": DB_USERNAME,
+        "password": DB_PASSWORD,
+        "prefix": "",
```

Comment:
Create a new file for this. Maybe `orator_production.py`

Reply:
I let them all be in the same orator.py. Tell me what you think.